kafka-connect/kafka-connect-runtime/docker/docker-compose.yml

```diff
@@ -50,6 +50,7 @@ services:
     environment:
       - AWS_REGION=us-east-1
       - CATALOG_WAREHOUSE=s3://bucket/warehouse/
+      - CATALOG_URI=jdbc:sqlite:file:/tmp/iceberg_rest_mode=memory
```

Comment:
Maybe the testcase is written such a way that, tables needs to be persisted. 
Hence, it is passing with file based URI and failing with default URI. 

Might have to refactor testcase. But locally I am not able to run these tests on MAC. 
So, using the workaround.